src/Chainweb/Version.hs

```diff
@@ -756,7 +744,6 @@ skipFeatureFlagValidationGuard
     -> BlockHeight
         -- ^ height of header
     -> Bool
-skipFeatureFlagValidationGuard Mainnet01 _ = True
-skipFeatureFlagValidationGuard Development _ = True
-skipFeatureFlagValidationGuard Testnet04 _ = True
+skipFeatureFlagValidationGuard Mainnet01 h = h < 530500  -- ~ 2020-05-01T00:00:xxZ
+skipFeatureFlagValidationGuard Development h = h < 140
```

Comment:
This causes validation failures. It's here only for testing the respective mainnet behavior. Those tests have been completed by now.

```suggestion
```